### PR TITLE
MBS-11481 (MBS-10847): Add refine link to edit/subscribed 

### PR DESCRIPTION
**Pull request by reosarevok**
### Implement MBS-11481

Also add a missing refinement for subscribed editor edits.

### CODE REVIEW

File: lib/MusicBrainz/Server/Controller/Edit.pm [hunk 1]
```diff
@@ -320,6 +320,21 @@ sub subscribed : Local RequireAuth {
     $c->stash(
         edits => $edits, # stash early in case an ISE occurs
         template => 'edit/subscribed.tt',
+        refine_url_args =>
+            { auto_edit_filter => '', order=> 'asc', negation=> 0,
+              combinator=>'and',
```
[the remark this answers is not in the document]
Comment (reosarevok) [reply lib/MusicBrainz/Server/Controller/Edit.pm:325]: This seems to be the same for all `refine_url_args`, fixing the rest in a separate commit.

**Comment by jesus2099** [comment]
Do you think it would be possible to have a more natural date than [_Created after **867600 seconds ago**_](https://beta.musicbrainz.org/search/edits?auto_edit_filter=&order=asc&negation=0&combinator=and&conditions.0.field=edit_subscription&conditions.0.operator=subscribed&conditions.1.field=open_time&conditions.1.operator=%3E&conditions.1.args.0=867600+seconds+ago&conditions.1.args.1=&conditions.2.field=editor&conditions.2.operator=not_me&conditions.2.name=&conditions.2.args.0=&conditions.3.field=voter&conditions.3.operator=me&conditions.3.name=&conditions.3.voter_id=&conditions.3.args=no&field=Please+choose+a+condition)?

**Comment by reosarevok** [comment]
The best possible equivalent is `241 hours ago` it seems, because `10 days and 1 hour ago` is not accepted. Not sure if that's a lot better, but maybe a bit?

**Comment by jesus2099** [comment]
It's slightly better, yes, I did not notice it meant 10 days and 1 hour.
The best would be if we could change it to `10 days` by getting rid of that extra 1 hour. 

But `241 hours` is already better than seconds.

**Comment by reosarevok** [comment]
I think the point of the 1 hour is that 10 days and 59 minutes is the max time an edit can be open (7 days plus 3 days of extension with a No vote plus 1 hour until ModBot applies it) :)

**Comment by jesus2099** [comment]
And should it be bad idea to use `11 days`?
I'm just thinking out loud. 🤔

**Pull request merged**